### PR TITLE
Add diagram showing ideal kit assembly

### PR DESCRIPTION
**Pull request by Adimote**
Adds a diagram showing the ideal assembly for kit, the image was `pngcrush`-ed

image

### CODE REVIEW

**Review by trickeydan — APPROVED** [review]
This is really really good Andy!

**Review by raccube — APPROVED** [review]

**Comment by WillB97** [comment]
Looks great, but we seem to be missing the USB with competitors code on.

File: kit/assembly.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ title: Kit Assembly
 Kit Assembly
 ============
 
-The Student Robotics kit contains a number of separate modules which must be connected together to be usable to control your robot.
+The Student Robotics kit contains a number of separate modules which must be connected together to be usable to control your robot. Below is a diagram of what your assembled kit should look like.
```
Comment (WillB97) [inline kit/assembly.md:9]: Something didn't flow right.
```suggestion
The Student Robotics kit contains a number of separate modules which must be connected together to control your robot. Below is a diagram of what your assembled kit should look like.
```

**Comment by sedders123** [comment]
Sorry for the drive by comment, is there a reason we are using a PNG rather than a minified SVG?

**Comment by Adimote** [comment]
> Sorry for the drive by comment, is there a reason we are using a PNG rather than a minified SVG?

I didn't see other SVGs in the website and was scared of  integration hell being the first to put one in

**Comment by raccube** [comment]
You can just use an SVG like a regular image in an `` or `<picture>` tag so I don't see why not.

**Pull request merged**